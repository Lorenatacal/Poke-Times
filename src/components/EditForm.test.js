import React, { Component } from 'react';
import EditFormConnected, { EditForm } from './EditForm';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

Enzyme.configure({ adapter: new Adapter() });

test('EditForm should render correctly', () => {
    const props = {
        post : {
            title: 'First post',
            body: 'First body',
        }
    }
    const wrapper = Enzyme.shallow(<EditForm {...props}/>)
    expect(toJson(wrapper)).toMatchSnapshot();
})