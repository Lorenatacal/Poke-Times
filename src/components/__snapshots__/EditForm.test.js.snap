// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditForm should render correctly 1`] = `
<div
  className="center post card"
>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      placeholder="Your title"
      value="First post"
    />
    <textarea
      onChange={[Function]}
      placeholder="Insert the post body"
      value="First body"
    />
    <button
      onClick={[Function]}
      type="Submit"
    >
      Submit 
    </button>
  </form>
</div>
`;
